db/crud.go

```diff
@@ -1852,12 +1852,14 @@ func (db *Database) updateAndReturnDoc(docid string, allowImport bool, expiry ui
 			_shallowCopyBody: storedDoc.Body(),
 		}
 		db.revisionCache.Put(documentRevision)
+
 		if db.EventMgr.HasHandlerForEvent(DocumentChange) {
 			webhookJSON, err := doc.BodyWithSpecialProperties()
 			if err != nil {
 				base.Warnf("Error marshalling doc with id %s and revid %s for webhook post: %v", base.UD(docid), base.UD(newRevID), err)
 			} else {
-				err = db.EventMgr.RaiseDocumentChangeEvent(webhookJSON, docid, oldBodyJSON, revChannels)
+				winningRevChange := newRevID == doc.CurrentRev
```

Comment:
Does this cover the case where a new tombstone revision makes an already existing revision the active revision? I think this would be considered a change in the active revision.  i.e.
1. Doc is in conflict, has 3-a (winning) and 3-b
2. New revision 4-a tombstones 3-a, 3-b is now the winning revision
3. doc.CurrentRev=3-b, but I'm not sure newRevID=3-b.

Reply:
Thanks for catching. Expanded test to cover these and fixed.